src/fields.jl

```diff
@@ -0,0 +1,64 @@
+include("grids.jl")
+
+export Grid1D, Grid2D
+
+
```

Comment:
Delete these lines. No need to include them again. All these are part of the `MixedLayerThermoclineDynamics` module (from the main file in the `src` directory) so the grids have been included and the `Grid1D/2D` have already been exported.

Reply:
I see. I'll delete them.